mlrun/frameworks/sklearn/mlrun_interface.py

```diff
@@ -97,7 +97,7 @@ def mlrun_fit(cls):
 
         def wrapper(
             self: SKLearnTypes.ModelType,
-            X: SKLearnTypes.DatasetType,
+            X: SKLearnTypes.DatasetType,  # noqa: N803 - should be lowercase "x", kept for BC
```

Comment:
tbh I understand that this is a pep rule but there is general convention in which `X` represents machine learning inputs (either train set or test set). For instance, take a look at https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

Reply:
I see your point, but it should be used primarily on examples/tutorials and not as a public API.
If there's a need for it - a "noqa" waiver can be added.